CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
     - respect muted channels
     - respect muted users
     - decrement when message is hard deleted
+- Fix paginated channels in channel list were left without channels when sync is executed [#1985](https://github.com/GetStream/stream-chat-swift/issues/1985)
```

Comment:
```suggestion
- Fix paginated channels in channel list were left without messages when sync is executed [#1985](https://github.com/GetStream/stream-chat-swift/issues/1985)
```
```suggestion
- Fix paginated channels in channel list were left without channels when sync is executed [#1985](https://github.com/GetStream/stream-chat-swift/issues/1985)
```

Reply:
😅 good catch. Will fix it when rebasing with Nuno's on the changelog